Frontend/ui-customer/src/Components/DriverInfoCard/Common/View.purs

```diff
@@ -291,49 +291,68 @@ sourceDestinationView push config =
   let isNotRentalRide = (config.fareProductType /= FPT.RENTAL)
       bottomMargin = (if os == "IOS" && config.rideStarted && config.enablePaddingBottom then (safeMarginBottom + 36) else 12)
       dropLocationTextWidth = if config.enableEditDestination && isNotRentalRide then if os == "IOS" then V ((screenWidth unit) / 100 * 80) else V ((screenWidth unit) / 100 * 65) else V ((screenWidth unit) / 10 * 8)
+      pickupLocationTextWidth = if config.isEditPickupEnabled && config.rideAccepted then if os == "IOS" then V ((screenWidth unit) / 100 * 80) else V ((screenWidth unit) / 100 * 65) else V ((screenWidth unit) / 10 * 8)
```

Comment:
@Shikhar-24  can we change the code to something like this. 
abc = V ((screenWidth unit) / 100 * 80) else V ((screenWidth unit) / 100 * 65)
xyz = V ((screenWidth unit) / 10 * 8)
if config.isEditPickupEnabled && config.rideAccepted then if os == "IOS" then abc else xyz